docs/updating/from_1.x_2.x/update_db_to_2.5.md

```diff
@@ -71,12 +71,16 @@ To use it:
     from `ez_systems_landing_page_field_type` under `ezplatform_page_fieldtype` in the new config.
     Otherwise the script will encounter errors.
 
-
 You can remove the bundle after the migration is complete.
 
-The command migrates Landing Pages created in eZ Platform v1.x, v2.0 and v2.1 to new Pages.
+The `ezplatform:page:migrate` command migrates Landing Pages created in eZ Platform v1.x, v2.0 and v2.1 to new Pages.
 The operation is transactional and rolls back in case of errors.
 
+!!! caution "Avoid exception when migrating from eZ Publish"
```

Comment:
Why do we actually need this section in the migration *from v1.13*?

Reply:
The ticket mentions 2.5.14  as the affected version, and in current documentation the entry point to 2.5.x is from 1.13. At a certain point of their update path to Ibexa Platform users would end up  at this stage and this is where the update goes south. Hopefully they have read the migration guide but if they did not, this is the last warning before the failure.

Reply:
OK, so I'd reformulate the next sentence to something like "If you migrated to v1.13 from eZ Publish..." and so on. Because they are not _migrating_ from eZ Publish _now_, but might have done it before.